documents/Specification/MaterialX.Proposals.md

```diff
@@ -188,7 +188,27 @@ We have a standard 3d fractal noise, but a 2d variant would be useful as well.
     * `period` (float or vector3): the positive integer distance at which the noise function returns the same value for input coordinate repeated at that step.  Default is 0, meaning the noise is not periodic.
     * `in` (float): the 1D coordinate at which the noise is evaluated.
 
+<a id="node-worleynoise2d"> </a>
 
+Expanded 2D Worley noise to support different distance metrics and periodicity. 
+
+* **`worleynoise2d`**: 2D Worley noise using centered jitter, outputting float (distance metric to closest feature), vector2 (distance metrics to closest 2 features) or vector3 (distance metrics to closest 3 features).
+    * `metric` (uniform string): the distance metric to return, one of "distance" (Euclidean distance to feature), "distance2" (Euclidean distance squared), "manhattan" or "chebyshev".  Default is "distance".
+    * `jitter` (float): amount to jitter the cell center position, with smaller values creating a more regular pattern.  Default is 1.0.
```

Comment:
I think you intended to carry over `period` from the spec to the proposals doc - but it looks like `jitter` and `texcoord` came along for the ride and `period` got dropped? 

similarly below...

Reply:
Woh, how did I do that... Thanks for catching it. I did intend to include the existing parameters, to give a more "complete" picture? I thought that was the style, but I'll narrow it down for clarity sake. Thanks @ld-kerley